### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by BuddahLD**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2015-2018, Virgil Security, Inc.
```
Comment (andrii-iakovenko) [inline build.gradle]: 2015-2020

Comment (BuddahLD) [reply build.gradle]: all copyrights are updated in second PR

File: pythia/src/test/java/com/virgilsecurity/pythia/ConfigurableTest.java [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  /**
+   * Get Pythia service base URL.
+   *
+   * @return Pythia service base URL.
+   */
+  public String getPythiaServiceUrl() {
+    return this.propertyReader.getProperty(PYTHIA_SERVICE_ADDRESS);
+  }
+
+//  /**
```
Comment (andrii-iakovenko) [inline pythia/src/test/java/com/virgilsecurity/pythia/ConfigurableTest.java]: Outdated and commented code

File: pythia/src/test/java/com/virgilsecurity/pythia/ProofKeysTest.java [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+/**
+ * Unit tests for {@link ProofKeys}.
+ * 
+ * @author Andrii Iakovenko
+ *
+ */
+public class ProofKeysTest {
+
+//  private SampleDataHolder sample;
```
Comment (andrii-iakovenko) [inline pythia/src/test/java/com/virgilsecurity/pythia/ProofKeysTest.java]: Outdated and commented code

File: pythia/src/test/java/com/virgilsecurity/pythia/PythiaContextTest.java [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+/**
+ * Unit test for {@link PythiaContext}.
+ * 
+ * @author Andrii Iakovenko
+ *
+ */
+public class PythiaContextTest {
+
+//  private String appId;
```
Comment (andrii-iakovenko) [inline pythia/src/test/java/com/virgilsecurity/pythia/PythiaContextTest.java]: Outdated and commented code

File: pythia/src/test/java/com/virgilsecurity/pythia/PythiaExamplesTest.java [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+/**
+ * Integration test which covers documentation snippets.
+ * 
+ * @author Andrii Iakovenko
+ *
+ */
+public class PythiaExamplesTest extends ConfigurableTest {
+
+//  @Test
```
Comment (andrii-iakovenko) [inline pythia/src/test/java/com/virgilsecurity/pythia/PythiaExamplesTest.java]: Outdated and commented code

File: pythia/src/test/java/com/virgilsecurity/pythia/client/VirgilPythiaClientTest.java [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+
+/**
+ * Integration tests for {@link VirgilPythiaClient}.
+ * 
+ * @author Andrii Iakovenko
+ *
+ */
+public class VirgilPythiaClientTest extends ConfigurableTest {
+
+//  private PythiaCrypto pythiaCrypto;
```
Comment (andrii-iakovenko) [inline pythia/src/test/java/com/virgilsecurity/pythia/client/VirgilPythiaClientTest.java]: Outdated and commented code

**Pull request merged**